mupen64plus-core/upstream/src/main/main.c

```diff
@@ -557,8 +557,21 @@ void main_state_inc_slot(void)
     savestates_inc_slot();
 }
 
+void main_state_load_latest_auto_save()
+{
+    if (netplay_is_init() || l_FileName == NULL)
+        return;
+    savestates_set_job(savestates_job_load, savestates_type_unknown, l_FileName);
+}
+
 void main_state_load(const char *filename)
 {
+
+    if(filename != NULL) {
+        l_FileName = malloc(strlen(filename) + 1);
```

Comment:
I see a malloc here, but where is the free?

Reply:
The free is on line 1926 when all the plugins are closed from the user exiting, do you think it could leak out some other ways?